components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/test/resources/dbScripts/h2.sql

```diff
@@ -79,26 +79,54 @@ CREATE TABLE IF NOT EXISTS IDN_OAUTH2_ACCESS_TOKEN (
             SUBJECT_IDENTIFIER VARCHAR(255),
             ACCESS_TOKEN_HASH VARCHAR (512),
             REFRESH_TOKEN_HASH VARCHAR (512),
+            IDP_ID INTEGER DEFAULT -1 NOT NULL,
+            TOKEN_BINDING_REF VARCHAR (32) DEFAULT 'NONE',
             PRIMARY KEY (TOKEN_ID),
             FOREIGN KEY (CONSUMER_KEY_ID) REFERENCES IDN_OAUTH_CONSUMER_APPS(ID) ON DELETE CASCADE,
             CONSTRAINT CON_APP_KEY UNIQUE (CONSUMER_KEY_ID,AUTHZ_USER,TENANT_ID,USER_DOMAIN,USER_TYPE,TOKEN_SCOPE_HASH,
-                                           TOKEN_STATE,TOKEN_STATE_ID)
+                                           TOKEN_STATE,TOKEN_STATE_ID,IDP_ID,TOKEN_BINDING_REF)
 );
 
-CREATE INDEX IDX_AT_CK_AU ON IDN_OAUTH2_ACCESS_TOKEN(CONSUMER_KEY_ID, AUTHZ_USER, TOKEN_STATE, USER_TYPE);
```

Comment:
Is there a reason why we removing and adding new tables?

Reply:
The old database did not have some required tables related to the Federated authentication. So I used an existing database which contained all required tables.